sdk/python/kubeflow/trainer/test.Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Use an official Python runtime as a parent image
```

Comment:
I think, we can name this file Dockerfile.cpu, similar to how we do for Katib: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/trial-images/pytorch-mnist

Reply:
Got it! I will update it.